### PR TITLE
Add all metadata elements to registration

### PR DESCRIPTION
**Pull request by beckyjackson**
I've updated the registration template to include all the metadata elements we were previously getting from https://ihccglobal.org/membercohorts/

image

I also added some data validation to (hopefully) make it easier for people to fill out, for example:

image

... and ...

image

These data items will now be stored in [data/metadata.json](https://github.com/IHCC-cohorts/data-harmonization/blob/admin-metadata/data/metadata.json). We can convert back to the CSV format to give to IHCC global:
```
make build/member_cohorts.csv
```

Right now this only includes the cohorts we have, but we definitely can add all the others (we will need to assign them IDs if we do this, though).

The metadata is sorted by the key, which is the cohort ID instead of the cohort name, so it rearranged the order of some things (which is why the diff is a little messy). The only real change in the cohort data should be the addition of `irb_approved_data_sharing`. This data comes from the admin metadata, and we wanted to add it in #134 as well. It's the only data item in the current admin metadata that we didn't already have in our JSON.

### CODE REVIEW

**Comment by mcourtot** [comment]
Sounds amazing!

File: src/prefixes.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
         "CMO": "http://purl.obolibrary.org/obo/CMO_",
         "COB": "http://purl.obolibrary.org/obo/COB_",
         "EFO": "http://purl.obolibrary.org/obo/EFO_",
+        "FOO": "https://purl.ihccglobal.org/FOO_",
```
Comment (jamesaoverton) [inline src/prefixes.json:9]: Was this just for testing, or is it used somewhere?

Comment (beckyjackson) [reply src/prefixes.json]: Sorry, that was a test. I removed it.

**Comment by beckyjackson** [comment]
@mcourtot how quickly do we want to implement this? Do we want the new cohorts that John emailed to be filling out this new sheet, or should we continue to get the metadata from IHCC global?

**Comment by beckyjackson** [comment]
@rosibaj this PR updates the cohort JSON - https://raw.githubusercontent.com/IHCC-cohorts/data-harmonization/admin-metadata/data/cohort-data.json

It adds `irb_approved_data_sharing` with values true or false (as discussed in #134). This might only be relevant for the DAC version of the atlas, but maybe that's something we can talk about on the next call? There should not be any other changes, but please let me know if you see a problem with the JSON.

**Review by jamesaoverton — APPROVED** [review]

**Pull request merged**